tests/e2e_tests/subcommands/root/test_root_boost_weights.py

```diff
@@ -0,0 +1,92 @@
+from bittensor.commands.root import (
+    RootSetBoostCommand,
+    RootSetWeightsCommand,
+    RootRegisterCommand,
+)
+from bittensor.commands.stake import StakeCommand
+from bittensor.commands.network import RegisterSubnetworkCommand
+from bittensor.commands.register import RegisterCommand
+from ...utils import (
+    setup_wallet,
+    sudo_call_set_network_limit,
+    sudo_call_set_weight_limit,
+    sudo_call_set_min_stake,
```

Comment:
All of these `sudo_call`'s need to be replaced with Btcli commands.

Reply:
> All of these `sudo_call`'s need to be replaced with Btcli commands.

It would be very helpful if you could leave a link to a similar test case that has already been implemented